src/utils/db.utils.ts

```diff
@@ -66,7 +66,16 @@ export function getWhere<T extends typeof BaseModel>(model: T, filterString?: st
         operator = `${operator} ${value}`;
         value = null;
       }
-
+      if (operator.startsWith('ILIKE') || operator.startsWith('LIKE')) {
+        const operatorSplit = operator.split(' ');
+
+        if (operatorSplit.length > 1) {
+          // Handle cases where {value} is a text that includes a space
+          // ie "ILIKE Ash Ketchup"
+          operator = operatorSplit[0];
+          value = operatorSplit.slice(1).join(' ');
+        }
+      }
```

Comment:
A fix has to be generic as we support all `SqlWhereOperator`s. Each of these is a valid test case:
`name = 'John Smith'`
`name NOT LIKE '% Smith'`
`name IS NOT NULL`
`name IN ('Adam','Bruce','Craig')`
`age = 30`

Also note the `buildFilter` method. All string values should be surrounded by single quotes and all arrays surrounded by parentheses.